### DISCUSSION TITLE
v2.5.0

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#80] [Announcements]
feat: support using premium account to watch lives(NicoNico)
feat: support changing resolutions of videos(NicoNico)
feat: support watching premium videos(NicoNico)

fix: crash when importing existed backups

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/AnimePipe/releases/tag/'>v2.5.0</a>.</em>

**Comment by pipcollector** [comment]
Can you please add "download" in drop down menu when long press a video in What's New?

**Reply by InfinityLoop1308** [reply]
tracked by #87 . Will possibly be supported in the next version.

**Comment by hydroakri** [comment]
Can you add patreon or afdian link？I really hope this project wont lose maintanence

**Reply by InfinityLoop1308** [reply]
Thank you so much for your feedback and your interest in supporting AnimePipe! I'm glad to hear that you find it great. 

At this time, I'm not too concerned about money as my primary limitation is actually time. However, I truly appreciate your willingness to support me financially. 

If you'd like to help out in other ways, I would love to hear any suggestions you may have to improve the project or any bugs you may have encountered while using it. Your feedback is invaluable to me, and it's what helps me make the project better for everyone.
